Pod/SQL/GDGEntity+SQL.h

```diff
@@ -0,0 +1,34 @@
+//
+//  GDGEntity+SQL.h
+//  GoldDigger
+//
+//  Created by Felipe Lobo on 5/4/16.
+//
+
+#import "GDGEntity.h"
+
+@class SQLEntityMap;
+@class SQLEntityQuery;
+
+@interface GDGEntity (SQL)
+
+@property (strong, nonatomic) NSMutableArray *filledProperties;
+@property (strong, nonatomic) NSMutableArray *changedProperties;
+
++ (SQLEntityMap *)db;
```

Comment:
Well, it is a concept issue. Once we start to use inheritance in a more straight manner, does not make sense.